crates/swc_ecma_codegen/src/lib.rs

```diff
@@ -75,6 +75,56 @@ where
     pub wr: W,
 }
 
+fn replace_close_inline_script(raw: &str) -> Cow<str> {
+    let chars = raw.as_bytes();
+    let pattern_len = 8; // </script>
+    let matched = |i: usize| {
+        if i + pattern_len >= chars.len() {
+            return false;
+        }
+        chars[i] == b'<'
+            && chars[i + 1] == b'/'
```

Comment:
This will panic for `</scr`

Reply:
Why do you think that? Lines 82-84 have performed a boundary check